doc/elements.md

```diff
@@ -105,6 +105,29 @@ def file(data):
 
 If `file()` does not return anything, it will be treated as if `file()` returned the empty string.
 
+## `pl_file_upload` element
+
+```html
+<pl_file_upload files="foo.py, bar.c, filename with\, comma]" />
```

Comment:
Our convention for name-like attributes is to call them things like `answers_name`, `params_name`, etc. You might consider using `file_name` as the attribute here instead of `files`.